File: app/src/main/java/org/mesonet/app/formulas/DefaultUnits.java
package org.mesonet.app.formulas;



public interface DefaultUnits
{
    UnitConverter.TempUnits GetDefaultTempUnit();
    UnitConverter.LengthUnits GetDefaultLengthUnit();
    UnitConverter.SpeedUnits GetDefaultSpeedUnit();
    UnitConverter.PressureUnits GetDefaultPressureUnit();
}

File: app/src/main/java/org/mesonet/app/formulas/DefaultUnits.kt
package org.mesonet.app.formulas


interface DefaultUnits {
    fun GetDefaultTempUnit(): UnitConverter.TempUnits?
    fun GetDefaultLengthUnit(): UnitConverter.LengthUnits?
    fun GetDefaultSpeedUnit(): UnitConverter.SpeedUnits?
    fun GetDefaultPressureUnit(): UnitConverter.PressureUnits?
}
